.github/workflows/registry.yml

```diff
@@ -44,6 +44,9 @@ jobs:
           registry: docker.io
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Login Alibaba Cloud ACR
+        run: |
```

Comment:
Replace it with https://github.com/marketplace/actions/alibaba-cloud-container-registry-acr-login